.github/workflows/build-test.yml

```diff
@@ -212,7 +212,7 @@ jobs:
           cd tools
           cargo run --bin generate-fuzz-corpus
       - name: build
-        run: cargo fuzz list | xargs -I "%" sh -c "cargo fuzz run % -- -max_total_time=30 || exit 255"
+        run: cargo fuzz list | xargs -I "%" sh -c "cargo fuzz run % -- -max_total_time=30 || exit 255  --all-features"
```

Comment:
If you make the fuzzer depend on `clvmr/counters` I don't think you would need this